### PR TITLE
Fix + cleanup broken build issues.

### PR DESCRIPTION
**Pull request by bendory**

### CODE REVIEW

**Comment by bendory** [comment]
Note: this PR fixes the breakage in the use of the native `git_repository` rule. build step still has a build failure in target `:go_context_data`.

File: bazel/cloudbuild.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # In this directory, run the following command to build this builder.
-# $ gcloud builds submit . --config=cloudbuild.yaml
+# $ gcloud builds submit
```
Comment (Philmod) [inline bazel/cloudbuild.yaml]: no need for the source / config?

File: bazel/.gcloudignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+cloudbuild.yaml
```
Comment (Philmod) [inline bazel/.gcloudignore]: why is that?

File: bazel/cloudbuild.yaml (hunk 1 above)
Comment (bendory) [reply bazel/cloudbuild.yaml]: No need -- the default values are no ``config=cloudbuild.yaml` and `.`

File: bazel/.gcloudignore (hunk 2 above)
Comment (bendory) [reply bazel/.gcloudignore]: `gcloud` reads the `cloudbuild.yaml` and converts it into the build request; there is no need to submit it to the `/workspace` for the build.

**Pull request merged**